csv.go

```diff
@@ -142,7 +142,7 @@ func (cp *csvPrinter) printStart(hostname string, port uint16) {
 func (cp *csvPrinter) printProbeSuccess(localAddr string, userInput userInput, streak uint, rtt float32) {
 	hostname := userInput.hostname
 	if hostname == "" {
-		hostname = "-"
+		hostname = ""
```

Comment:
the `if` statement is unnecessary.

Reply:
oops, yeah I did a find and replace. Thanks for pointing that out 😅